docs/config/layouts/styles/header.css

```diff
@@ -144,6 +144,90 @@ header nav {
   text-decoration: none;
 }
 
+.version-dropdown {
+  position: relative;
+  display: flex;
+  font-family: "Open Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;
+  font-size: 14px;
+  text-decoration: none;
+  margin-right: var(--space-unit);
+  height: 30px;
+}
+
+.version-dropdown-toggle {
+  background: none;
+  border: none;
+  color: #ffffff;
+  cursor: pointer;
+  padding: 10px 0;
+  display: flex;
+  align-items: center;
+}
+
+.version-dropdown-toggle:after {
+  content: "";
+  border-top: 5px solid #fff;
+  border-right: 5px solid transparent;
+  border-left: 5px solid transparent;
+  display: inline-block;
+  margin-left: 5px;
+  vertical-align: middle;
+}
+
+
+.version-dropdown-toggle:hover,
+.version-dropdown-toggle:focus {
+  color: #e3e1e1;
+}
+
+.dropdown-header {
+  font-weight: 600;
+  text-align: left;
+  display: flex;
+  align-items: center;
+  height: 44px;
+  padding-left: 10px;
+  color: #a2a2a2;
+
+}
+
+.version-dropdown-menu {
+  display: none;
+  position: absolute;
+  top: 100%;
```

Comment:
So this `top:100%` is a bit odd.

Reply:
That is to prevent the dropdown menu from vanishing while hovering over the "gap".

Reply:
Eh it's fine.